src/cmem/markdown/Markdown.tsx

```diff
@@ -2,6 +2,7 @@ import ReactMarkdown from "react-markdown";
 import {PluggableList} from "react-markdown/lib/react-markdown";
 import rehypeRaw from "rehype-raw";
 import remarkGfm from "remark-gfm";
+//@ts-ignore
```

Comment:
i did not have a good idea on how to deal with the missing typings in this package. How is this solved in the other UIs?